www/heatmap.html

```diff
@@ -0,0 +1,236 @@
+                        <span class="sr-only">Toggle navigation</span>
+                        <span class="icon-bar"></span>
+                        <span class="icon-bar"></span>
+                        <span class="icon-bar"></span>
+                    </button>
+                    <a class="navbar-brand" href="https://webgazer.cs.brown.edu/">WebGazer.js</a>
+                </div>
+                <div id="navbar" class="navbar-collapse collapse">
+                    <ul class="nav navbar-nav nav-tabs" role="tablist">
+                        <li><a href="https://webgazer.cs.brown.edu/#home">Home</a></li>
+                        <li><a href="https://webgazer.cs.brown.edu/#video">About</a></li>
+                        <li><a href="https://webgazer.cs.brown.edu/#usage">Usage</a></li>
+                        <li><a href="https://webgazer.cs.brown.edu/#support">Support</a></li>
+                        <li><a href="https://webgazer.cs.brown.edu/#download">Download</a></li>
+                        <li><a href="https://webgazer.cs.brown.edu/documentation">Documentation</a></li>
+                        <li class="active"><a href="https://webgazer.cs.brown.edu/#examples">Examples</a></li>
+                        <li><a href="https://webgazer.cs.brown.edu/#publication">Publication</a></li>
+                        <li><a href="https://webgazer.cs.brown.edu/#press">Press</a></li>
+                        <li><a href="https://webgazer.cs.brown.edu/#contact">Contact</a></li>
+                    </ul>
+                </div>
+            </div>
+        </nav>
+
+        <div id="heatmap-home" class="container pad" style="padding-top: 0px">
+            <div class="jumbotron">
+                <h2 class="text-center" style="margin-bottom:0px">Data Visualization</h2><br>
+                <div class="text-center"><a href="https://www.patrick-wied.at/static/heatmapjs/">Heatmap.js</a>, created by <a href="https://www.patrick-wied.at">Patrick Wied,</a> is a "lightweight, easy to use JavaScript library to help you visualize your three dimensional data!"</div>
+            </div>
+
+            <hr class="featurette-divider">
+            <h2 class="text-center" style="margin-bottom:0px">Example Demo</h2><br>
+            <p class="text-center padright" style="font-size:20px">This demo does not work with an ad-blocker.</p>
+
+            <div class="row text-center">
+                <div class="col-lg-4">
+                    <form method="get" action="https://www.patrick-wied.at/static/heatmapjs/docs.html">
+                        <button class="btn btn-primary btn-lg text-center" type="submit">Documentation</button>
+                    </form>
+                </div>
+               <div class="col-lg-4">
+                    <form method="get" action="./data/heatmap-tutorial.zip">
```

Comment:
This zip heatmap-tutorial.zip isn't pushed to the repository and I get the error "Cannot GET /data/heatmap-tutorial.zip?" when I press this button